_posts/for-developers/apps/vocabularies/2019-01-01-01_quickstart-creation.md

```diff
@@ -1,14 +1,16 @@
 ---
 layout: for-developers
-title: "Create your vocabulary "
+title: "How to create your own vocabulary"
 permalink: /for-developers/apps/vocabularies/create/quickstart
 tags: [apps]
 categories: [Vocabularies, Quickstart]
 exclude: true
 published: true
 ---
 
-Let us build a very simple vocabulary describing [obelisks](https://en.wikipedia.org/wiki/Obelisk), so that [Cleopatra](https://cleopatra.solid.community/profile/card#me) and [Caesar](https://jcaesar.solid.community/profile/card#me) can share information about their personal collections.
+# How to create your own vocabulary
+
+Let's build a very simple vocabulary describing [obelisks](https://en.wikipedia.org/wiki/Obelisk) (tall, four-sided, narrow tapering monuments that end in a pyramid-like shape), so that [Cleopatra](https://cleopatra.solid.community/profile/card#me) and [Caesar](https://jcaesar.solid.community/profile/card#me) can share information about their personal collections.
```

Comment:
Great example, but I think it should mention that there tons of vocabs, and that in most cases, you'd just use somebody else's rather than create your own, it is even considered best practice.

Reply:
Absolutely: this is just a 5-minutes intro to RDF, which I focused on things you can do quickly and on your own. However, the "in-depth" tutorial (see `2019-01-01-00_overview.md` for the overview) opens on a list of existing vocabularies, and gives some hints about how to search for a vocabulary, exactly for the reason you mention.